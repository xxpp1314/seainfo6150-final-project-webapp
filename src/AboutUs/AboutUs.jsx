import React from 'react';
import styles from "./AboutUs.module.css";
import AboutUs from "../images/AboutUs.jpg";
import { Link } from "react-router-dom";

const AboutUS = () => {
    return (            
        <div className={styles.mainpage}>
            <div className={styles.grid}>
                    <div className={styles.flex}>
                        <div className={styles.flexitems}>
                            <img className={styles.image} src={AboutUs} alt=""></img>
                        </div>
                        <div className={styles.gap}> </div>
                        <div className={styles.flexitems}>
                            <h1>A Little Bit About Us</h1><br /><br />
                            <p>In North America millions of lost, stray, and abandoned animals enter shelters every year. With so many animals available, pet adoption is quickly becoming the most popular method of finding a new pet.</p><br />
                            <p>However, pets are innocent and adorable. So, If you are considering adopt a pet, we are the place that can help you find your suitable pet.</p><br />
                            <p>In addition, if you have ideas or suggestions to improve our service, please feel free to speak your thought. Click <Link className={styles.text} to="/feedback">here</Link> to provide your feedback</p><br /><br />
                        </div>
                    </div>
            </div>   
        </div>
    )
}

export default AboutUS;