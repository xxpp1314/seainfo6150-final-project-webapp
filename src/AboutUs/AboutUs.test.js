import React from "react";
import { render } from "@testing-library/react";
import AboutUs from "./AboutUs.jsx";
import { BrowserRouter } from 'react-router-dom';

it("renders the AboutUs Page correctly", () => {
  const { container } = render(
    <BrowserRouter>  
        <AboutUs/>
    </BrowserRouter> 
  );
  expect(container).toMatchSnapshot();
});