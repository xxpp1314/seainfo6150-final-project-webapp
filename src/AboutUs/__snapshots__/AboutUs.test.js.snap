// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the AboutUs Page correctly 1`] = `
<div>
  <div
    class="mainpage"
  >
    <div
      class="grid"
    >
      <div
        class="flex"
      >
        <div
          class="flexitems"
        >
          <img
            alt=""
            class="image"
            src="AboutUs.jpg"
          />
        </div>
        <div
          class="gap"
        >
           
        </div>
        <div
          class="flexitems"
        >
          <h1>
            A Little Bit About Us
          </h1>
          <br />
          <br />
          <p>
            In North America millions of lost, stray, and abandoned animals enter shelters every year. With so many animals available, pet adoption is quickly becoming the most popular method of finding a new pet.
          </p>
          <br />
          <p>
            However, pets are innocent and adorable. So, If you are considering adopt a pet, we are the place that can help you find your suitable pet.
          </p>
          <br />
          <p>
            In addition, if you have ideas or suggestions to improve our service, please feel free to speak your thought. Click 
            <a
              class="text"
              href="/feedback"
            >
              here
            </a>
             to provide your feedback
          </p>
          <br />
          <br />
        </div>
      </div>
    </div>
  </div>
</div>
`;
